tools/run_command.py

```diff
@@ -21,15 +21,11 @@
         parser.print_help()
         exit(1)
 
-    try:
-        if args.subcommand == 'tidy':
-            check_tidy.run_check_tidy(args)
-        elif args.subcommand == 'test':
-            integration_test.run_integration_test(args)
-        elif args.subcommand == 'build':
-            build_example.run_build_examples(args)
-        elif args.subcommand == 'plugins':
-            print_plugins.run_print_plugins(args)
-    except Exception as e:
-        print(e)
-        exit(1)
```

Comment:
I'm fine that you remove this,  but I want to know a reason to remove them.

Reply:
Now that illegal cli arguments are all checked between lines 19~22, the try-catch block will only catch exceptions that's meaningful for debugging purposes only. (The tool is not for end-user developers so I'm not differentiating programming errors and runtime exceptions.) Catching exceptions will only print the error message while letting the program crash will let the python interpreter to print stack trace so it's easy to debug and exit with code 1 automatically.